bin/run.py

```diff
@@ -354,9 +340,13 @@ def process_run(run, p):
             color = Fore.GREEN if self.verdict in self.expected_verdicts else Fore.RED
             boldcolor = ''
 
+        max_duration = max(d for d in verdicts.duration.values() if d is not None)
+        salient_testcase = verdicts.salient_testcase()
         printed_newline = bar.finalize(
-            message=f'{max_duration:6.3f}s {color}{self.print_verdict:<20}{Style.RESET_ALL} @ {verdict_run.testcase.name}'
+            message=f'{max_duration:6.3f}s {color}{self.verdict:<20}{Style.RESET_ALL} @ {salient_testcase}'
```

Comment:
Instead of separately printing the `max_duration` and the name of the `salient_testcase`, shouldn't we print the time of the `salient_testcase` (since the printed time and name ought to be connected)? :thinking: The advantage here is, that the salient test case happens to be the one with the max duration anyway (at least, in the case that the root verdict is AC) :smile: